### PR TITLE
update nv endpoints

### PR DESCRIPTION
**Pull request by NVJCameron**

### CODE REVIEW

**Comment by juberti** [comment]
Hi, thanks for the pull request. Are these -turbo models full precision or quantized?

**Comment by NVJCameron** [comment]
> Hi, thanks for the pull request. Are these -turbo models full precision or quantized?

These endpoints are FP8.

**Comment by juberti** [comment]
ah ok. In that case please use LLAMA_3_8B_CHAT_FP8.

Also, it looks like all Nvidia models were removed in the last commit, was that intentional?

**Comment by NVJCameron** [comment]
No it was not - it was a botched merge from upstream. I will correct. Thanks!

**Review by juberti — APPROVED** [review]

File: llm_benchmark_suite.py [hunk 1]
```diff
@@ -460,11 +458,8 @@ def _tools_models():
         #    "accounts/fireworks/models/llama-v3p1-405b-instruct", LLAMA_31_405B_CHAT_FP8
         # ), returns "FUNCTION" and the call as text
         _GroqLlm("llama-3.1-405b-reasoning", LLAMA_31_405B_CHAT_FP8),
-        _NvidiaLlm("meta/llama-3.1-405b-instruct", LLAMA_31_405B_CHAT),
```
[the remark this answers is not in the document]
Comment (NVJCameron) [reply llm_benchmark_suite.py:463]: Yes I meant to remove them. Thanks for checking!

[the remark this answers is not in the document]
Comment (juberti) [reply llm_benchmark_suite.py:463]: OOC why?

[the remark this answers is not in the document]
Comment (NVJCameron) [reply llm_benchmark_suite.py:463]: There are still some optimizations to be done for tool usage. Will add them back at a later time.

**Pull request merged**